### PR TITLE
mobile rubric arrow button interactions

### PR DESCRIPTION
**Pull request by semibran**
Rally: [US126937](https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fuserstory%2F521010078100)

Follow-up from #782, part of the inline grading rubric revamp

Fixes selected criterion levels in place for preview/graded mode and improves overall screen reader usability.

### CODE REVIEW

File: d2l-rubric-criterion-mobile.js [hunk 1]
```diff
@@ -263,6 +282,8 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-rubric-criterion-mobile">
 				</div>
 			</template>
 		</div>
+
+		<p class="criterion-status sr-only" role="status" aria-live="polite"></p>
```
Comment (semibran) [inline d2l-rubric-criterion-mobile.js:286]: Decentralizing screen reader functionality from aria-valuetext seems to be a necessary evil in order to get the arrow buttons to tie in properly

File: d2l-rubric-levels-mobile.js [hunk 2]
```diff
@@ -329,25 +332,57 @@ Polymer({
...
+			case this._keyCodes.UP:
+				if (slider.value === upperBound) {
+					criterion.say(this.localize('noMoreLevels'));
+				}
+				break;
+		}
 	},
 
-	_onKeyDown: function(e) {
+	_handleKeyUp: function(e) {
```
Comment (semibran) [inline d2l-rubric-levels-mobile.js:363]: would've liked to merge these into a single listener, but it looks like there's some bounds checking nuances involved for preview/graded mode

**Comment by MitchelPaulin** [comment]
Make sure to update all lang terms https://github.com/Brightspace/d2l-rubric#lang-term-update

File: d2l-rubric-criterion-mobile.js [hunk 3]
```diff
@@ -373,15 +399,45 @@ Polymer({
 		return this.CriterionAssessmentHelper.getCompetencyNames(assessmentCriterionEntity);
 	},
 
+	say: function(text) {
+		const status = this.shadowRoot.querySelector('.criterion-status');
+		status.textContent = text;
+		setTimeout(() => {
+			status.textContent = '';
+		}, 1000);
```
[the remark this answers is not in the document]
Comment (MitchelPaulin) [reply d2l-rubric-criterion-mobile.js:407]: I dont think so, unfortunately we don't have much control over what the screen reader does, we can only try to coerce it.

This is also a function where I think a comment for why it exists would be helpful

File: d2l-rubric-criterion-mobile.js (hunk 1 above)
Comment (MitchelPaulin) [reply d2l-rubric-criterion-mobile.js:286]: From what I understand this is to get the screenreader to read out text properly. Have you considered using the `d2l-offscreen` component?

Comment (semibran) [reply d2l-rubric-criterion-mobile.js:286]: Completely slipped my mind, thanks for the suggestion

**Pull request merged**